static/css/index.css

```
@@ -47,13 +47,14 @@ label[for=max-clicks] {
 }
 
 .alias-time-container .child {
-    width: 100%; /* FireFox Fallback */
-    width: -webkit-fill-available;
+    width: -webkit-fill-available; /* FireFox Fallback */
+    width: 100%;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)